codalab/apps/web/migrations/0087_auto__add_field_competitiondefbundle_competition.py

```diff
@@ -0,0 +1,503 @@
+# -*- coding: utf-8 -*-
```

Comment:
we shouldn't need like 4-5 migrations here? can just be one? i guess maybe too late since this was already deployed.. but we should avoid deploying so many migrations when they could be squashed together?

Reply:
Yes, apologies. After meeting with Isabelle, we decided to implement some more features and I had to add some more migrations.

I think it's possible we could still squash them, LRI is the only server with this currently deployed for testing (on a branch separate from lri-master so we can always rollback). We could rollback migrations to `0089` for web, and then on redeploy with squashed migrations it should be fine.